src/components/templates/Jazzbar/JazzBar/JazzBar.tsx

```diff
@@ -166,29 +67,15 @@ export const JazzBar: React.FC<JazzProps> = ({ venue }) => {
         </div>
       )}
 
-      {/*seatedAtTable && (
```

Comment:
`DISABLED_DUE_TO_` check or delete entirely, having a commented code just makes it invisible to tools thus shift away from current code